corgi/tasks/sca.py

```diff
@@ -86,58 +108,37 @@ def save_component(
     if component["type"] not in Component.Type.values:
         raise ValueError("Tried to save component with unknown type: %s", component["type"])
 
-    component_type = component["type"].lower()
-    syft_purl = meta.get("purl")
-    if not syft_purl:
-        # TODO: Needs a lot more work and tests
-        # But this fixes the most basic issues for now
-        qualifiers = {}
-
-        version = meta.get("version", "")
-        release = meta.get("release")
-        if version and release:
-            version = f"{version}-{release}"
-
-        arch = meta.get("arch")
-        if arch:
-            qualifiers["arch"] = arch
-        # TODO: Might need a epoch qualifier
-
-        syft_purl = PackageURL(
-            type=component_type,
-            name=meta.get("name", ""),
-            version=version,
-            qualifiers=qualifiers,
-        ).to_string()
-
     created = False
     name = meta.pop("name", "")
+    # TODO: Do we need to add epoch to the unique constraint?
```

Comment:
Probably yes. As long as it's always default to the same value for components without an epoch. Should be done in a separate PR / Task of course.